src/transactions.ts

```diff
@@ -1,10 +1,30 @@
 import { TransactionStatus } from "./transactionStatus";
 import { ContractResults } from "./contractResults";
 import { Address } from "./primitives";
-import { IAddress } from "./interface";
+import { IAddress, ITransactionNext } from "./interface";
 import { TransactionLogs } from "./transactionLogs";
 import { TransactionReceipt } from "./transactionReceipt";
 
+export function prepareTransactionNextForBroadcasting(transaction: ITransactionNext): any {
```

Comment:
Perhaps also do a `perpareTransactionForBroadcasting(legacy | next)` and have here (only in one place) the check about having "toSendable"?

Reply:
Done